### PR TITLE
add freetype CI

### PR DESCRIPTION
**Pull request by borune-k12**
Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by borune-k12** [comment]
Rerun action after #61 will be merged

File: .github/workflows/freetype.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+name: freetype CI
+
+on:
+  pull_request:
+    paths:
+      - 'recipes/freetype/**'
```
Comment (worldemar) [inline .github/workflows/freetype.yml]:
```suggestion
      - 'recipes/freetype/all/**'
```

File: .github/workflows/freetype.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      - 'build.py'
+      - '.github/workflows/freetype.yml'
+  push:
+    branches:
+      - master
+    paths:
+      - 'recipes/freetype/**'
```
Comment (worldemar) [inline .github/workflows/freetype.yml]:
```suggestion
      - 'recipes/freetype/all/**'
```

File: .github/workflows/freetype.yml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+env:
+  CONAN_PASSWORD: ${{ secrets.BintrayApiKey }}
+  IS_PURE_C: true
+
+jobs:
+  freetype:
+    strategy:
+      matrix:
+        build: ['linux-gcc4.8', 'linux-gcc7', 'linux-gcc9', 'windows-2019-release', 'windows-2019-debug', 'macos']
+        freetype_versions: ['2.10.0', '2.10.1']
```
Comment (Artalus) [inline .github/workflows/freetype.yml]: нам реально нужны два пакета, отличающиеся на единичку в минорной версии? или там как с OpenSSL?

Comment (borune-k12) [reply .github/workflows/freetype.yml]: Смотри. Если смотреть по конан-файлам, то для Qt нужен 2.10.0, а для fontconfig нужен 2.10.1.

**Pull request merged**